packages/visx-text/package.json

```diff
@@ -40,8 +40,5 @@
   },
   "publishConfig": {
     "access": "public"
-  },
-  "devDependencies": {
-    "@testing-library/react-hooks": "^8.0.0"
```

Comment:
left over from https://github.com/airbnb/visx/pull/1890, hoping this fixes the failing `@visx/react-spring` hooks test suite